qbosdk/apis/api_base.py

```diff
@@ -85,7 +85,7 @@ def _query_get_all(self, object_type: str, url: str) -> List[Dict]:
 
         raise QuickbooksOnlineSDKError('Error: {0}'.format(response.status_code), response.text)
 
-    def _get_request(self, object_type: str, api_url: str) -> List[Dict] or Dict:
```

Comment:
Why remove object_type, are you sure it won't break anything in the apps?

Reply:
Yes, this change won't break anything in the apps
object_type is not needed, the API response is same even after removing that param